microshift_storage/container_storage_interface_microshift/microshift-persistent-storage-csi.adoc

```diff
@@ -0,0 +1,29 @@
+:_content-type: ASSEMBLY
+[id="persistent-storage-csi-microshift"]
+= Configuring CSI volumes for MicroShift
+include::_attributes/attributes-microshift.adoc[]
+:context: persistent-storage-csi-storage-microshift
+
+toc::[]
+
+The Container Storage Interface (CSI) allows {product-title} to consume
+storage from storage back ends that implement the
+link:https://github.com/container-storage-interface/spec[CSI interface]
+as persistent storage.
+
+[NOTE]
+====
+{product-title} {product-version} supports version 1.5.0 of the link:https://github.com/container-storage-interface/spec[CSI specification].
+====
+
+include::modules/persistent-storage-csi-architecture.adoc[leveloffset=+1]
+
+include::modules/persistent-storage-csi-external-controllers.adoc[leveloffset=+2]
+
+include::modules/persistent-storage-csi-driver-daemonset.adoc[leveloffset=+2]
+
+include::modules/persistent-storage-csi-drivers-supported.adoc[leveloffset=+1]
```

Comment:
This module doesn't fit with the actual design of MicroShift.  Namely, it doesn't install any of the CSI drivers on the list.